app/views/trainees/review_draft/show.html.erb

```diff
@@ -75,76 +75,76 @@
     <h2 class="govuk-heading-m">About their teacher training</h2>
 
     <%= render TaskList::View.new(classes: "record-setup") do |component|
-          if show_publish_courses?(@trainee)
```

Comment:
I didn't make any actual changes to this file - it's just correcting indentation.